pkgs/development/libraries/openmpi/default.nix

```diff
@@ -66,13 +68,13 @@ in stdenv.mkDerivation rec {
     # https://github.com/openucx/ucx
     # https://www.open-mpi.org/faq/?category=buildcuda
     ++ lib.optionals cudaSupport [ "--with-cuda=${cudatoolkit_joined}" "--enable-dlopen" ]
-    ++ lib.optionals fabricSupport [ "--with-psm2=${libpsm2}" "--with-libfabric=${libfabric}" ]
+    ++ lib.optionals fabricSupport [ "--with-psm2=${lib.getDev libpsm2}" "--with-libfabric=${lib.getDev libfabric}" ]
     ;
 
   enableParallelBuilding = true;
 
   postInstall = ''
-    rm -f $out/lib/*.la
+    find $out/lib/ -name "*.la" -exec rm -f \{} \;
```

Comment:
Unrelated change. Please put it in a separate commit and explain the what/why of that change.

(I checked and found that there is only one *.la file in the output: `result/lib/openmpi/libompi_dbg_msgq.la`. So that means the current code is a NOP. I'm not sure why we're removing *.la files though.)

Reply:
It actually has la files in `lib/` and the one in `lib/openmpi` (the old version just removed them):
```
result/lib/libmpi.la
result/lib/libmpi_usempif08.la
result/lib/libompitrace.la
result/lib/openmpi/libompi_dbg_msgq.la
result/lib/libmpi_usempi_ignore_tkr.la
result/lib/libmpi_mpifh.la
result/lib/liboshmem.la
result/lib/libopen-pal.la
result/lib/libmpi_cxx.la
result/lib/libopen-rte.la
```

A far as I am aware, these libtool files are not needed for anything on Linux (or on Darwin). The idea here is to simply to reduce the output size/number of files in the output.

Reply:
Hehe, silly me, the rm code worked :-)